includes/functions.php

```diff
@@ -365,51 +365,29 @@ function ( $matches ) {
 function is_activitypub_request() {
 	global $wp_query;
 
-	/*
-	 * ActivityPub requests are currently only made for
-	 * author archives, singular posts, and the homepage.
-	 */
-	if ( ! \is_author() && ! \is_singular() && ! \is_home() && ! defined( '\REST_REQUEST' ) ) {
-		return false;
-	}
-
-	// Check if the current post type supports ActivityPub.
-	if ( \is_singular() ) {
-		$queried_object = \get_queried_object();
-		$post_type      = \get_post_type( $queried_object );
-
-		if ( ! \post_type_supports( $post_type, 'activitypub' ) ) {
-			return false;
-		}
-	}
-
-	// Check if header already sent.
-	if ( ! \headers_sent() && ACTIVITYPUB_SEND_VARY_HEADER ) {
-		// Send Vary header for Accept header.
-		\header( 'Vary: Accept' );
-	}
-
 	// One can trigger an ActivityPub request by adding ?activitypub to the URL.
 	if ( isset( $wp_query->query_vars['activitypub'] ) ) {
 		return true;
 	}
 
 	/*
-	 * The other (more common) option to make an ActivityPub request
-	 * is to send an Accept header.
-	 */
+	* The other (more common) option to make an ActivityPub request
+	* is to send an Accept header.
+	*/
```

Comment:
```suggestion
	 * The other (more common) option to make an ActivityPub request
	 * is to send an Accept header.
	 */
```
Was this an IDE setting? They should stay aligned